.github/workflows/meterpreter_acceptance.yml

```diff
@@ -203,18 +228,43 @@ jobs:
           bundler-cache: true
           cache-version: 4
           working-directory: metasploit-framework
-          # Github actions with Ruby requires Bundler 2.2.18+
-          # https://github.com/ruby/setup-ruby/tree/d2b39ad0b52eca07d23f3aa14fdf2a3fcc1f411c#windows
-          bundler: 2.2.33
 
       - name: Move mettle gem into framework
-        if: ${{ matrix.meterpreter.name == 'mettle' && (contains(github.event.issue.labels.*.name, 'mettle-testing-branch')) }}
+        if: ${{ matrix.meterpreter.name == 'mettle' && contains(github.event.pull_request.labels.*.name, 'payload-testing-mettle-branch') }}
         run: |
           cp ../mettle/pkg/metasploit_payloads-mettle-${{ env.METTLE_VERSION }}.pre.dev.gem .
         working-directory: metasploit-framework
 
+      - uses: actions/download-artifact@v4
+        name: Download Java meterpreter
+        id: download
```

Comment:
Why an `id`, and why not `java-meterpreter-download` instead of `download`?

Reply:
`id` is required to reference the result later as a variable; have updated the value :+1: